docs/install.md

```diff
@@ -60,13 +60,13 @@ Once the ANTLR sources have been generated, you can compile the entire project l
 
     $ mvn clean compile assembly:single
     
-After successful completion, you can check the `target` directory, which should contain the compiled classes as well as the JAR file `jsoniq-spark-app-0.9.7-jar-with-dependencies.jar`.
+After successful completion, you can check the `target` directory, which should contain the compiled classes as well as the JAR file `jsoniq-spark-app-1.0-jar-with-dependencies.jar`.
```

Comment:
Shouldn't the jar name here be the same as the line below?
spark-rumble-1.0-jar-with-dependencies.jar

Reply:
Good catch!